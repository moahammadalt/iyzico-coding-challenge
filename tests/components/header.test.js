import React, { Component } from 'react';
import { BrowserRouter as Router, Link } from "react-router-dom";
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import Header from '../../src/components/header';


// Snapshots
describe('<Header /> Snapshot test for Header component', () => {

  it('Should match the Header snapshot', () => {

    const wrapper = shallow(
      <Router>
        <Header />
      </Router>
    );

    const tree = renderer.create(wrapper).toJSON();

    expect(tree).toMatchSnapshot();
  });

});