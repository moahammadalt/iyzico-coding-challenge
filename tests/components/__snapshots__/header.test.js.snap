// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Snapshot test for Header component Should match the Header snapshot 1`] = `
<header
  className="header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <h1
      className="logo"
    >
      RICK AND MORTY
    </h1>
  </a>
</header>
`;
